### PR TITLE
[RFR] Fixed test cases of applications page

### PR DESCRIPTION
**Pull request by ganeshhubale**
Signed-off-by: Ganesh Hubale <ghubale@redhat.com>
- Fixed test case : `test_applications_page`, `test_add_applications_to_project`

### CODE REVIEW

File: mta/entities/applications.py [hunk 1]
```diff
@@ -24,10 +24,9 @@
 class ApplicationsView(BaseLoggedInPage):
 
     title = Text(locator=".//div/h2[normalize-space(.)='Applications']")
-    add_application_button = Button("Add")
-    upload_file = HiddenFileInput(id="fileUpload")
-    search = Input("searchValue")
-    close_search = Text(locator=".//span[@class='pficon pficon-close']")
+    add_application_button = Text(locator='.//button[contains(text(), "Add application")]')
```
Comment (sshveta) [inline mta/entities/applications.py:27]: Changing button Import to patternfly 4 will fix the problem .

     from widgetastic_patternfly4 import Button
     add_application_button = Button("Add application")

File: mta/entities/applications.py [hunk 2]
```diff
@@ -44,7 +43,8 @@ def is_displayed(self):
     def clear_search(self):
         """Clear search"""
         if self.search.value:
-            self.close_search.click()
+            self.search.fill("")
+            self.browser.refresh()
```
Comment (sshveta) [inline mta/entities/applications.py:47]: Is browser refresh necessary ?

**Review by sshveta — CHANGES_REQUESTED** [review]

File: mta/entities/applications.py (hunk 2 above)
Comment (ganeshhubale) [reply mta/entities/applications.py:47]: yes. Only filling input with `" "`  does not show application.
browser refresh helps to regain the old position.

File: mta/entities/applications.py (hunk 1 above)
Comment (ganeshhubale) [reply mta/entities/applications.py]: fixed!

**Review by sshveta — APPROVED** [review]

**Pull request merged**